coalib/coala_modes.py

```diff
@@ -1,4 +1,14 @@
 def mode_normal(console_printer, log_printer, args, debug=False):
+    """
+    This is the default coala mode. User interaction is allowed in this mode
+
+    :param console_printer: Object to print messages on the console.
+    :param log_printer:     A LogPrinter object to use for logging.
+                            It is now deprecated and unused.
```

Comment:
Because it's deprecated, I would actually just say `:param log_printer: Deprecated.`